Demo/UITests/BraintreeDropIn_UITests.swift

```diff
@@ -138,6 +138,11 @@ class BraintreeDropIn_securityCodeValidation_CardForm_UITests: XCTestCase {
         securityCodeField.forceTapElement()
         securityCodeField.typeText("200")
 
+        let postalCodeField = elementsQuery.textFields["12345"]
```

Comment:
The "Add Card" button is not hittable until all fields are filled so we needed to fill in the postal code element for this test